README.md

```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Vehicles App
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React
+- Redux
 
-### `npm start`
+## Getting Started
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+To get a local copy up and running follow these simple example steps.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- copy this link `https://github.com/gids-dadi/metrics-webapp`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `https://github.com/gids-dadi/metrics-webapp`.
+- go to the repository folder in your command prompt `metrics-webapp`.
+- Open terminal in that folder and run `npm run start`.
 
-### `npm test`
+## Live link
```

Comment:
- [ ] Please deploy properly the application using Heroku, Netlify, or GitHub Pages.
- [ ]
![1](https://user-images.githubusercontent.com/63972670/176998151-d7005ac6-95f1-40bf-bf6a-8f761b53d898.PNG)